Sources/GRPCCodeGen/Internal/StructuredSwiftRepresentation.swift

```diff
@@ -31,6 +31,11 @@
 ///
 /// For example: `import Foo`.
 struct ImportDescription: Equatable, Codable {
+  /// The access level of the imported module.
+  ///
+  /// For example, the `public` in `public import Foo`.
+  var accessLevel: AccessModifier
```

Comment:
This should be optional.

You should think about building this from the other direction: you should start with the build blocks, i.e. the 'structured swift' and build from there. Starting there it's more obvious that this should be optional: because you don't have to have explicit import access levels.

It's only when you get to the grpc code gen layer that we decide to be explicit about the imports.

Doing this also means we can have less churn in the tests as all the existing tests for import description would remain as valid tests. You'd only need to add one more test with an explicit access level to make sure our bases our covered.

Reply:
I had started with this as an optional (both here and in `ImportDescription`), but because by the time I got to the higher layers and was always being explicit about it, I thought it added some unnecessary logic/unwraps elsewhere.

I'm not sure I understand the "It's only when you get to the grpc code gen layer that we decide to be explicit about the imports." point. Both this `StructuredSwiftRepresentation` and the `ImportDescription` are at the `GRPCCodeGen` layer - that's precisely why I ended up changing my mind about it. 

Having said that, I can make it optional again: I can see the argument for it from a "Swift doesn't require you to have it, so shouldn't we" perspective. I was just trying to simplify the logic if we'll always have a value for it in our codegen.

Reply:
> I'm not sure I understand the "It's only when you get to the grpc code gen layer that we decide to be explicit about the imports." point. Both this `StructuredSwiftRepresentation` and the `ImportDescription` are at the `GRPCCodeGen` layer - that's precisely why I ended up changing my mind about it.

You're conflating the module and layer here. You can have different API layers within a module and that's what we have here. 

The surface API of the module is the code gen request and response. Within the module there are separate API layers: a translation layer which coverts the request into structured swift, and a rendering layer which coverts the structured swift into text. 

It's the translation layer that decides to be explicit about imports, the rendering layer doesn't express an opinion on whether they should have access levels, it just knows how to render them.

Making it optional is a smaller change (so reduces churn) and gives us more flexibility. If, for example, we need to add an option about whether generated code should have explicit imports then making that change would be easy: we'd express that opinion in the request and pass it to the translation layer.